RecoMuon/MuonIdentification/interface/MuonShowerDigiFiller.h

```diff
@@ -46,6 +46,7 @@
 
 class MuonShowerDigiFiller {
 public:
+  MuonShowerDigiFiller() {}
```

Comment:
thanks for checking.
I forgot that the implicit default is created only if there is no other kind defined.